catalog-spec/catalog-spec.md

```diff
@@ -116,14 +116,49 @@ details on the two types and how you might use them see the [Static and Dynamic
 | Element      | Type          | Description                                                  |
 | ------------ | ------------- | ------------------------------------------------------------ |
 | stac_version | string        | **REQUIRED.** The STAC version the catalog implements.       |
+| stac_extensions | [string]   | A list of extensions the catalog and its items have implemented. |
```

Comment:
Wasn't this going on the collection and item levels?

Reply:
Not sure about this. We should check use cases and see where we need it.